.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,93 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        stable: false
+        go-version: 1.15
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Install golangci-lint
+      uses: golangci/golangci-lint-action@fix-75
```

Comment:
the fix-75 branch was merged into master, but not yet released. You can change fix-75 to master I think